src/include/cursor.h

```diff
@@ -211,33 +211,30 @@ struct __wt_cursor_btree {
 struct __wt_cursor_bulk {
 	WT_CURSOR_BTREE cbt;
 
-	WT_REF	*ref;			/* The leaf page */
-	WT_PAGE *leaf;
```

Comment:
@agorrod:

> Was there a reason why you moved these lower in the structure?

The `ref`, `leaf` and `reconcile` fields are all part of reconciliation's support for bulk-load, so I put them in one place.